dags/map_reproducibility/utils.py

```diff
@@ -127,21 +130,100 @@ def copy_bucket_cmds():
 def get_metrics_cmds():
   # TODO(gunjanj007): get these parameters from the recipe
   get_metrics = (
+      "METRICS_FILE=$COMPLETE_JOB_NAME/metrics.txt",
       "python3 process_training_results.py --file"
       " dllogger.json --batch_size 2048 "
       "--num_accelerators 256 "
       "--precision fp8  "
       "--model_type gpt3-175b "
-      "--accelerator_type h100 ",
+      "--accelerator_type h100 | "
+      "gsutil cp - $METRICS_FILE",
+      'echo "METRICS_FILE=${METRICS_FILE}"',
   )
   return get_metrics
 
 
+def get_aotc_repo():
+  gob_clone_cmds = (
+      "echo 'trying to clone GoB aotc repo'",
+      "git clone https://cmcs-perf-tooling-internal.googlesource.com/"
+      "benchmark-automation",
+      "cd benchmark-automation/aotc/src",
+      "export PYTHONPATH=$PWD",
+      'echo "PYTHONPATH=$PYTHONPATH and METRICS_FILE=$METRICS_FILE"',
+  )
+  return gob_clone_cmds
+
+
 def cleanup_cmds():
   cleanup = (
-      "kubectl get pods "
-      "--no-headers=true | awk '{print $1}' "
-      "| grep $JOB_NAME |  xargs kubectl delete pods",
-      "helm uninstall $JOB_NAME",
+      "cd $REPO_ROOT",
+      "cd ../../..",
+      # "kubectl get pods "
+      # "--no-headers=true | awk '{print $1}' "
+      # "| grep $JOB_NAME |  xargs kubectl delete pods",
+      # "helm uninstall $JOB_NAME",
   )
   return cleanup
+
+
+def get_metrics_from_gcs(bucket_name, file_name):
+  # bucket_name = 'gunjanjalori-testing-xlml'
+  # file_name = 'nemo-experiments/gpt3-xlml-1731373474-175b-nemo-1731373494-ic5n/metrics.txt'
```

Comment:
can we remove the comments for debugging?

Reply:
done